github.xverizex.nem-desktop.json

```diff
@@ -0,0 +1,41 @@
+{
+    "app-id" : "github.xverizex.nem-desktop",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "41",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "nem-desktop",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--socket=pulseaudio"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "nem-desktop",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/xverizex/nem-desktop.git",
```

Comment:
this is repository isn't found. Did you mean `https://github.com/xverizex/nem_desktop.git` ?

albeit it doesn't have a tag.

Reply:
I create a tag via the website. Help me with a valid app-id.

Reply:
https://github.com/xverizex/nem_desktop is valid